### ISSUE TITLE
tut dependency from bintray is flaky and causes issues on CI

### ISSUE THREAD

**Issue by ittaiz** [bazelbuild/rules_scala#580]
@philwo can we host this jar in the google mirror? If so how can we do this?

cc @johnynek

**Comment by johnynek** [comment]
yeah we are getting hit by this. Bintray is not happy somehow.

**Comment by tpolecat** [comment]
see https://github.com/tpolecat/tut/issues/210

**Comment by johnynek** [comment]
yeah we are getting hit by this internally. Bintray is not happy somehow.

**Comment by johnynek** [comment]
we need to revert #581 when this is solved.

**Comment by ittaiz** [comment]
@philwo are you the person who can help us with this? The reliance on this download site makes our Travis setup very flakey. We need this jar in a google mirror.
Btw, What we’d love for a way for rule maintainers to add jars there (maybe with a review policy).

**Comment by philwo** [comment]
We can host files on our Google Cloud Storage mirror, yes. Could you let me know the exact URL that you want to mirror? It's currently a manual process, but long-term I'd like to make it automatic and easy to use for others.
